### DISCUSSION TITLE
Apply Tailwind class to Input text

### DISCUSSION THREAD

**Discussion by clabnet** [lljj-x/vue-json-schema-form#197] [Q&A] [answered]
On this uiSchema code snippet :
```
 machinery: {
    "ui:options": {
      disabled: false,
      style: {
        boxShadow: '0 0 12px 12px #2b9939'
      },
      class: {
        className_hei: true
      },
    },
  },
```
how to apply a Tailwind class to machinery field  without use `style` ?
What is the className_hei ? 

Thanks in advance

**Comment by lljj-x** [comment, marked as answer]
Use the `class` option

Refer to vue render function：https://vuejs.org/guide/extras/render-function.html#declaring-render-functions .

 ```js
 machinery: {
    "ui:options": {
      disabled: false,
      style: {
        boxShadow: '0 0 12px 12px #2b9939'
      },

     // use Tailwind className
      class: ['mx-auto', 'flex'],
    },
  }
```

----

`className_hei` is just an example of a custom className
